src/test/java/steps/CreateStep.java

```diff
@@ -0,0 +1,102 @@
+package steps;
+
+import base.BaseUtil;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class CreateStep extends BaseUtil {
```

Comment:
Since you are using a `BaseUtil` instance as a field and not using the inherited methods, you should not extend it.